concent_api/concent_golem_messages/validators.py

```diff
@@ -0,0 +1,26 @@
+from golem_messages.exceptions import FieldError
+
+
+def validate_bytes(field_name: str, value: bytes):
+    assert isinstance(field_name, str)
+    assert isinstance(value, bytes)
+
+    if not isinstance(value, bytes):
+        raise FieldError(
+            'Should be a bytes field',
+            field=field_name,
+            value=value,
+        )
+
+
+def validate_maximum_int_length(field_name: str, value: int, maximum_allowed_length: int):
```

Comment:
Hmm, I just thought now, maybe better way to validate maximum int value (restricted by bytes, like in our example) could be:
```python
maximum_allowed_value = 2 ** (allowed_bytes * 8 ) - 1
if 2 ** (allowed_bytes * 8 ) - 1 < value:
   raise FieldError(
      f'Maximum allowed value is {maximum_allowed_value} but value is {value}',
      field=field_name,
      value=value,
   )
```
because for example if we assume that we can pass maximum 32 bytes = 256 bits it is possible to pass maximum 78 numbers, but also for 259 bits will have 78 numbers, look:
```bash
>>> len(str(2**(259)-1))
78
```
It is just proposal, but I think that might be better validator for this kind values to check :)

Reply:
I think that in the future we might want to validate other Concent golem-messages that will not necessary be based on bytes length, so I would skip that. But thanks to your comment I thought about something different; currently we reuse value `78` here and in `core`. How about placing it in `common/constants.py`, making it being calculated like in the code you have posted above, and describing what it describes ? 

What do you guys think ? This should be a separate issue.

Reply:
I see that right now it does this:
``` python
if maximum_allowed_length < len(str(value)):
```

I agree with @kbeker. Relying on string representation for comparing numbers is rarely a good idea. Does it even handle negative integers?

BTW, `if len(str(value)) > maximum_allowed_length:` reads much better as a condition. With the current one I have to go over it a few times to parse it mentally because of the weird order.